pkgs/applications/editors/zile/default.nix

```diff
@@ -11,9 +11,6 @@ stdenv.mkDerivation rec {
   buildInputs = [ pkgconfig ncurses boehmgc ];
   nativeBuildInputs = [ help2man perl ];
 
-  # `help2man' wants to run Zile, which fails when cross-compiling.
-  crossAttrs.nativeBuildInputs = [];
```

Comment:
Did you actually test that? IIRC coreutils suffers from the same problem, that to build a manpage for `foo`, it needs to execute `foo --help` piped to `help2man` which isn't obviously going to work when cross-compiling (and omitting help2man "helps" in the way that manpages aren't built and installed at all).

Reply:
I'm sorry @dezgeg. Indeed I had seen that same thing with coreutils. I forgot and thought the comment meant native `help2man` failed to build, which isn't true (and I checked that) but perhaps could have been when the `nativeDrv`s were weird.

I'll revert this part of it.

Reply:
Cross unfortunately fairly broken until https://github.com/NixOS/nixpkgs/pull/26805 or similar lands. My intent--which I failed at here--was to clean these things up without changing their behavior so I can more easily fix them later.